app/src/main/java/co/co/nimblehq/showcases/poly/place/ui/AppNavigation.kt

```diff
@@ -20,6 +22,14 @@ fun AppNavigation(
                 onGrantedLocationPermission = { /* TODO: Call Places SDK in https://github.com/nimblehq/poly-place/issues/32 */ }
             )
         }
+
+        composable(AppDestination.RestaurantDetails) {
+            RestaurantDetailsScreen(
+                navigator = { destination -> navController.navigate(destination) },
+                restaurant = navController.previousBackStackEntry?.savedStateHandle
+                    ?.get<Restaurant>(KeyRestaurant)
```

Comment:
Then here, we can use `Restaurant::class.simpleName` directly 🙏
```suggestion
                    ?.get<Restaurant>(Restaurant::class.simpleName)
```